packages/fx-core/src/plugins/resource/bot/dotnet/plugin.ts

```diff
@@ -59,13 +59,41 @@ export class DotnetBotImpl extends TeamsBotImpl {
     const appSettingsPath = path.join(context.root, PathInfo.appSettingDevelopment);
     try {
       let appSettings: string;
+      const includeAad = isAADEnabled(
+        context.projectSettings!.solutionSettings as AzureSolutionSettings
+      );
       if (await fs.pathExists(appSettingsPath)) {
         appSettings = await fs.readFile(appSettingsPath, "utf-8");
       } else {
         // if appsetting file not exist, generate a new one
         // TODO(qidon): load content from resource file or template
-        appSettings =
-          '\
+        appSettings = includeAad
+          ? '\
```

Comment:
maybe we should put these templates into a file?

Reply:
Agree, hard code the file content in code is not easy for maintenance, please change it with a better way.

Reply:
Sure, updated. @Siglud you may also need to update the appSettings without SSO part?